### PR TITLE
Exclude thrift module dependency from spring-boot-autoconfigure POM t…

### PR DESCRIPTION
**Pull request by imasahiro**
…o make a user can choose thrift version

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -560,6 +560,20 @@ configure(javaProjects) {
                                             return
                                         }
 
+                                        // Skip 'armeria-thrift-shaded' from armeria-spring-boot-autoconfigure's POM
+                                        // so that a user can choose between 'armeria-thrift(-shaded)' and
+                                        // 'armeria-thrift-0.9(-shaded).
+                                        if (currentArtifactId in ['armeria-spring-boot-autoconfigure', 
+                                                                  'armeria-spring-boot-autoconfigure-shaded'] &&
```
Comment (trustin) [inline build.gradle]: Could be just `currentArtifactId.startsWith('armeria-spring-')`?

Comment (imasahiro) [inline build.gradle]: Done

File: build.gradle [hunk 2]
```diff
@@ -560,6 +560,19 @@ configure(javaProjects) {
...
+                                        if (currentArtifactId.startsWith('armeria-spring-') &&
+                                            dep instanceof ProjectDependency &&
+                                            dep.dependencyProject.archivesBaseName in [
+                                                'armeria-thrift', 
+                                                'armeria-thrift-shaded']) {
+                                            logger.info "Excluding ${dep.group}:${dep.name} so that a user can choose thrift library version."
+                                            return
+                                        }
+
+
```
Comment (trustin) [inline build.gradle]: nit: double newlines

Comment (imasahiro) [inline build.gradle]: Removed

**Comment by trustin** [comment]
Thanks, @imasahiro !

**Pull request merged**